packages/cactus-plugin-ledger-connector-iroha/src/main/typescript/plugin-ledger-connector-iroha.ts

```diff
@@ -0,0 +1,684 @@
+import { Server } from "http";
+import * as grpc from "grpc";
+import { Server as SecureServer } from "https";
+import { CommandService_v1Client as CommandService } from "iroha-helpers-ts/lib/proto/endpoint_grpc_pb";
+import { QueryService_v1Client as QueryService } from "iroha-helpers-ts/lib/proto/endpoint_grpc_pb";
+import commands from "iroha-helpers-ts/lib/commands/index";
+import queries from "iroha-helpers-ts/lib/queries";
+//import RolePermission from "iroha-helpers-ts/lib/proto/primitive_pb";
+import type { Express } from "express";
+import { promisify } from "util";
+import { Optional } from "typescript-optional";
+import {
+  GrantablePermission,
+  GrantablePermissionMap,
+} from "iroha-helpers-ts/lib/proto/primitive_pb";
+//import { Contract, ContractSendMethod } from "web3-eth-contract";
+
+import {
+  ConsensusAlgorithmFamily,
+  IPluginLedgerConnector,
+  IWebServiceEndpoint,
+  IPluginWebService,
+  ICactusPlugin,
+  ICactusPluginOptions,
+} from "@hyperledger/cactus-core-api";
+
+import {
+  PluginRegistry,
+  consensusHasTransactionFinality,
+} from "@hyperledger/cactus-core";
+
+import {
+  Checks,
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+  Http405NotAllowedError,
+} from "@hyperledger/cactus-common";
+
+import {
+  IrohaCommand,
+  IrohaQuery,
+  // InvokeContractV1Request,
+  // InvokeContractV1Response,
+  RunTransactionRequest,
+  RunTransactionResponse,
+} from "./generated/openapi/typescript-axios";
+
+import { RunTransactionEndpoint } from "./web-services/run-transaction-endpoint";
+//import { isWeb3SigningCredentialNone } from "./model-type-guards";
```

Comment:
@hxlaf Please delete the commented lines unless there's a specific reason to keep them around.

Reply:
Fixed! Thank you!